cdbot/constants.py

```diff
@@ -54,6 +54,9 @@ class PostgreSQL:
     "Forensics": ["forensics", "f"],
 }
 
+# Admin Constants
+PLACEHOLDER_NICKNAME = "Valued server member"
+NICKNAME_PATTERNS = ["(discord.gg/)"]
```

Comment:
```suggestion
NICKNAME_PATTERNS = ["(discord\.gg/)"]
```
Dot matches any character (including but not limited to a literal dot). 

Though the more vague the regular expression the better so maybe this can be left as-is. (Ideally, we'd have a system where if the Levenchtein distance is 1 or 2 we'd still detect the nickname).

Reply:
We could use [`difflib.SequenceMatcher`](https://docs.python.org/3/library/difflib.html#sequencematcher-objects)? It's not Levenshtein, but it does do similarity ratios.

Reply:
![image](https://user-images.githubusercontent.com/25161069/56139867-a1be2300-5f91-11e9-9e10-37bfba7fbb99.png)

Reply:
Also the double backslash isn't needed in this case.